Src/DeallocateStoppedVM.ps1

```diff
@@ -1,10 +1,33 @@
-﻿Param
+﻿<#
+.SYNOPSIS 
+    This script deallocates every Azure virtual machines which are stopped.
+
+.DESCRIPTION
+    This script deallocates every Azure virtual machines which are stopped in the specified DevTest Lab.
```

Comment:
Expand comment. Read code and look at definition of Stopped (i.e. stopped but not deallocated).